### PR TITLE
small fixes

### LINKED ISSUE #36: Tags should not be prefixed with `#`

### PR DESCRIPTION
**Pull request by yahengsu**
fixes #36 and #34

### CODE REVIEW

File: server/discord.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ async function getPostsFromChannelMessages(channel) {
     try {
         //default fetch is 50 messages
         const channelMessages = await channel.messages.fetch();
-        const channelName = `#${channel.name}`;
+        const channelName = `${channel.name}`;
```
Comment (amy-lei) [inline server/discord.js]: Super super minor lol but now it’s unnecessary to use the `${}` syntax

**Review by amy-lei — APPROVED** [review]
Super minor thing but tbh dont even needa fix it. LGTM!!

File: server/discord.js (hunk 1 above)
Comment (yahengsu) [reply server/discord.js]: true lol

**Comment by yahengsu** [comment]
added #33

**Pull request merged**